transforms/setup-cli-build/__testfixtures__/test-with-plugin-property.input.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+
+module.exports = function (defaults) {
+  let app = new EmberApp(defaults, {
+    babel: {
+      plugins: [],
```

Comment:
Can we also add a test for when the plugins already contain an entry?

Reply:
> Can we also add a test for when the plugins already contain an entry?

Yeah, no problem